dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -132,4 +132,23 @@
       https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1296
     </Reason>
   </LinkageError>
+  <LinkageError>
+    <Source>
+      <Package name="io.netty.util.internal.logging" />
+    </Source>
+    <Reason>
+      io.netty.util.internal.logging.InternalLoggerFactory catches and handles errors caused by the
```

Comment:
Can probably delete io.netty.util.internal.logging. and just use the short name since it's in the Paclage statement above.

Reply:
Updated.